@here/harp-mapview/lib/text/RenderState.ts

```diff
@@ -119,6 +119,18 @@ export class RenderState {
         return this.m_state !== FadingState.FadedOut && this.m_state !== FadingState.Undefined;
     }
 
+    /**
+     * @returns `true` if state is neither faded out nor undefined and the opacity is larger
+     * than 0.
+     */
+    isReallyVisible(): boolean {
+        return (
+            this.m_state !== FadingState.FadedOut &&
+            this.m_state !== FadingState.Undefined &&
+            this.opacity > 0
+        );
+    }
+
```

Comment:
couldn't we instead check the opacity in isVisible() ?

Reply:
actually, shouldn't it always be the case that opacity > 0 for any states other than faded out or undefined?